### PR TITLE
Investment recipient company

### PR DESCRIPTION
**Pull request by ztolley**
Allow the user to search for and select a company the investment money will be sent to, stored as 'uk_company' in the back end.
uk company

### CODE REVIEW

File: src/apps/investment-projects/controllers/intermediate.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+const { searchCompanies } = require('../../search/services')
+const { transformCompanyToListItem } = require('../../companies/transformers')
+const { transformApiResponseToSearchCollection } = require('../../search/transformers')
+const { updateInvestment } = require('../repos')
+
+async function selectIntermediateResult (req, res, next) {
+  if (!req.query.intermediateCompany) {
```
Comment (jim68000) [inline src/apps/investment-projects/controllers/intermediate.js]: That’s not the right entity to save to. It’s supposed to be uk_company

The intermediate company is, amazingly, the intermediate not the final recipient

File: src/apps/investment-projects/controllers/ukcompany.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    )
+  } catch (error) {
+    return next(error)
+  }
+  next()
+}
+
+function renderCompanyResults (req, res, next) {
+  return res
+    .breadcrumb('Interactions')
```
Comment (jim68000) [inline src/apps/investment-projects/controllers/ukcompany.js]: Interactions?

File: src/apps/investment-projects/views/ukcompany.njk [hunk 3]
```diff
@@ -0,0 +1,30 @@
+{% extends "./_layout.njk" %}
+
+{% block main_grid_right_column %}
+  <h1 class="heading-medium">Recipient company</h1>
+
+  {{ EntitySearchForm({
+    inputName: 'term',
+    inputLabel: 'UK company that will receive the money',
```
Comment (jim68000) [inline src/apps/investment-projects/views/ukcompany.njk]: Can you check this with Mills? Or ignore me if you have

Comment (ztolley) [reply src/apps/investment-projects/views/ukcompany.njk]: Mills has said ok

Comment (jim68000) [reply src/apps/investment-projects/views/ukcompany.njk]: Cool

**Pull request merged**